classes/person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand1(1..1000)
```

Comment:
> - [ ] I noticed an error here that is also causing the project to produce incorrect results. It should be `rand(1..1000)` instead of `rand1(1..1000)`.
> 
> ```suggestion
>     @id = Random.rand(1..1000)
> ```